pkg/ocp/ocp.go

```diff
@@ -84,48 +92,67 @@ func DumpManifests(outputDir string, manifests ocp4.Manifests) {
 	}
 }
 
-func (config *ConfigMaster) Fetch(outputDir string) {
-	localF := filepath.Join(outputDir, config.Hostname, config.Path)
-	config.ConfigFile.Content = GetFile(config.Hostname, config.Path, localF)
-	logrus.Printf("File:Loaded: %s", localF)
-}
-
-func (config *ConfigNode) Fetch(outputDir string) {
-	localF := filepath.Join(outputDir, config.Hostname, config.Path)
-	config.ConfigFile.Content = GetFile(config.Hostname, config.Path, localF)
+func Fetch(configFile *ConfigFile) {
+	localF := filepath.Join(env.Config().GetString("OutputDir"), configFile.Hostname, configFile.Path)
+	configFile.Content = GetFile(configFile.Hostname, configFile.Path, localF)
```

Comment:
+1 for `GetFile`. I'm picturing a lot of the components requiring similar files so the fact that we have some caching logic is a good thing.